tests/driver_epd_bw_spi_disp_dev/Makefile

```diff
@@ -0,0 +1,9 @@
+include ../Makefile.tests_common
+
+USEMODULE += xtimer
+USEMODULE += epd_bw_spi
+USEMODULE += disp_dev
+
+INCLUDES += -I$(APPDIR)
```

Comment:
Same comment as above.

I also think that this application won't be technically required once the support for the stm32l0583-discovery on-board display will be added (I have a branch based on this PR for this): the default `tests/disp_dev` application will cover this use case.

Reply:
> I also think that this application won't be technically required once the support for the stm32l0583-discovery on-board display will be added

Note that I'm not asking to remove it here, just saying that it should be possible to reuse `tests/disp_dev` for the same purpose. And after looking a second time, some changes must be made in `tests/disp_dev` first, so can be done later.